frontend/lib/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -183,10 +184,8 @@ function PlotlyFigure({
       }
 
       return {
-        data: [...spec.data],
-        layout: {
-          ...spec.layout,
-        },
+        data: spec.data,
+        layout: spec.layout,
```

Comment:
I'm guessing that copying these was intentional to ensure that they don't get mutated down the line

Reply:
Hm I think I was actually writing that code before in the prototype and it got left in there so I was doing some minor clean up here.